Homework11

RomanKharitonov17 opened the pull request.

RomanKharitonov17 committed:
Добавление задания

RomanKharitonov17 committed:
Задание 1. Убрать цифры из текстового файла.

RomanKharitonov17 committed:
Задание 2. Логирование вызова функций

RomanKharitonov17 committed:
Задание 3. Три самые дорогие покупки.

RomanKharitonov17 committed:
Добавление заданий

RomanKharitonov17 committed:
Задание 2.

RomanKharitonov17 committed:
Задание 3.

RomanKharitonov17 committed:
Задание 4.

RomanKharitonov17 committed:
Задание 1.

RomanKharitonov17 committed:
Задание 2.

RomanKharitonov17 committed:
Задание 2.

RomanKharitonov17 committed:
Задание 4. Использование фикстуры

RomanKharitonov17 committed:
Задание 2. Дополнительная маркировка, маска positive

RomanKharitonov17 committed:
Задание 1. Декомпозиция логики генератора.
Добавление маркера в pytest.ini

RomanKharitonov17 committed:
Добавление заданий

RomanKharitonov17 committed:
Задание 1

RomanKharitonov17 committed:
Задание 1

RomanKharitonov17 committed:
Задание 2

vlad-vladimirovich commented on homework11/task_1.py:
```diff
@@ -0,0 +1,37 @@
+    time.sleep(1)
+    browser.get(tensor_url)
+    time.sleep(1)
+    block = browser.find_element(By.CSS_SELECTOR, '.tensor_ru-Index__block4-content')
```
а если номер блока поменяется, под этим номером может быть любой другой
можно объявить несколько элементов одним локатором и нужный тебе номер выбрать индексом

vlad-vladimirovich commented on homework11/task_1.py:
```diff
@@ -0,0 +1,37 @@
+    time.sleep(1)
+    browser.execute_script("arguments[0].scrollIntoView({behavior: 'auto', block: 'center'});", details)
+    time.sleep(3)
+    assert details.is_displayed()
```
ну да, у него есть атрибут ссылка, но это != тексту новости по сути проверка ничего не делает. хотелось бы вхождение текста проверить что бы знать а та ли эта новость

RomanKharitonov17 committed:
Update task_1.py

1. Поменял поиск на find_elements. Добавил выбор из списка
2. Добавил проверку текста ссылки.

RomanKharitonov17 committed:
Update task_1.py

Проверка названия блока "Сила в людях"
